database-mongo/index.js

```diff
@@ -13,22 +13,117 @@ db.once('open', function() {
 });
 
 var games = mongoose.Schema({
-  token: String,
-  players: String,
-  results: String  
+  token: {type: String, required: true},	
+  playerRoles: String,
+  results: String, 
+  playerIds: String,
 });
 
-var Game = mongoose.model('Game', games);
+Game = mongoose.model('Game', games);
```

Comment:
Why did the `var` go away? I'm not questioning if it is right, but asking *why* it is right.